src/python/espressomd/particle_data.pyx

```diff
@@ -692,11 +692,18 @@ cdef class ParticleHandle:
         # vs_auto_relate_to
         def vs_auto_relate_to(self, _relto):
             """
-            Setup this particle as virtual site relative to the particle with the given id.
+            Setup this particle as virtual site relative to the particle with the given ParticleHandle or id.
 
             """
-            check_type_or_throw_except(
-                _relto, 1, int, "Argument of vs_auto_relate_to has to be of type int.")
+            # If _relto is of type ParticleHandle,
```

Comment:
none of your business but this variable name is not too descriptive and the underscore is not pep8... just if your over-motivated :)

Reply:
I agree, the current variable names in this part of the interface are not as I would like them to be. However, this is true for all member variables of the `ParticleHandle` class. I think the idea here is to mirror the core variable but adding a prefixed underscore to not have the variables being **exactly** the same. I think @RudolfWeeber commented in some PR (that I can't find just know), that he would like to have distinct names for the variables in core and interface respectively. If I am wrong here, sure I can quickly go over the class and "pepify" it a bit.